include/cantera/kinetics/Kinetics.h

```diff
@@ -514,6 +514,14 @@ class Kinetics
         throw NotImplementedError("Kinetics::getDeltaSSEntropy");
     }
 
+    virtual void getDeltaRefEntropy(doublereal* deltaS) {
+        throw NotImplementedError("Kinetics::getDeltaRefEntropy");
+    }
+
```

Comment:
Please explain the functionality of `getDeltaRefGibss` and `getDeltaRefEntropy` with comments on top of the methods.